apps/marginfi-v2-ui/src/pages/api/oracle/price.ts

```diff
@@ -153,7 +153,7 @@ async function fetchCrossbarPrices(feedHashes: string[]): Promise<Map<string, Or
   const controller = new AbortController();
   const timeoutId = setTimeout(() => {
     controller.abort();
-  }, 5000);
+  }, 8000);
```

Comment:
Why did this change? It might be worth adding a comment to these hardcoded timeouts

Reply:
so I noticed the crossbar fetching sometimes times out to quickly causing the whole app to be stuck. I increased the abort a bit to see if I still get this. Especially if you have bad internet, pay attention to any instance when the app becomes stuck and pay attention to what caused it